.eslintrc.json

```diff
@@ -4,15 +4,27 @@
     "next",
     "next/core-web-vitals",
     "prettier",
-    "plugin:storybook/recommended"
+    "plugin:storybook/recommended",
+    "plugin:@typescript-eslint/recommended"
   ],
+  "parser": "@typescript-eslint/parser",
   "overrides": [
+    {
```

Comment:
What is this rule for?

Reply:
It was for `.mjs` files, removed it as I have moved `"plugin:@typescript-eslint/recommended"` line to the line no. 20

Reply:
Gotcha!